packages/amplication-notifications-service/src/contracts/notificationPattern.ts

```diff
@@ -0,0 +1,15 @@
+export class NotificationPattern {
```

Comment:
Likewise here, what is the value of making this a class rather than an interface?